api/src/wfl/service/datarepo.clj

```diff
@@ -196,10 +217,24 @@
       :id
       poll-job))
 
+;; By default, this endpoint retrieves all snapshot elements
```

Comment:
To me this TDR default behavior is weird, I asked about it here:

https://broadinstitute.slack.com/archives/CD4HBRFMG/p1643731101382549?thread_ts=1643398097.147800&cid=CD4HBRFMG